TWLight/users/tests.py

```diff
@@ -479,6 +479,38 @@ def test_my_library_page_has_authorizations(self):
         for partner in manual_partners:
             self.assertTrue(partner in response_manual_partners)
 
+        # Get collections
+        request = factory.get(reverse("users:redesigned_my_library"))
+        request.user = self.user_editor
+        response = views.MyLibraryView.as_view()(request)
+
+        # Check user_collection partner_id_set
+        user_auths = Authorization.objects.filter(user=self.user_editor).distinct()
+        test_partner_id_set = set()
+        for user_auth in user_auths:
+            if user_auth.is_valid:
```

Comment:
It occurs to me that this guard is incorrect on my part since you are intentionally including expired auths to offer renewals.

Reply:
I have added more tests that might check this. Let me know if those are OK!

Reply:
Yep, those tests look great to me. I realized I still needed to drop that guard clause because as is, this test only passes because it doesn't happen to setup expired apps.
```suggestion
        for user_auth in user_auths:
            for partner in user_auth.partners.all():
                test_partner_id_set.add(partner.pk)
```